cmd/chase.go

```diff
@@ -225,10 +230,10 @@ func parseStdInList(list []string, operating *string) (packages.IPackage, error)
 			"project_list": apkResult.ProjectList,
 		}).Trace("Obtained apk project list")
 		return apkResult, nil
-	case "fedora":
+	case "yum", "dnf":
 		logLady.WithFields(logrus.Fields{
 			"list": list,
-		}).Trace("Chasing Fedora")
+		}).Trace("Chasing dnf")
```

Comment:
In this case can we `log.Trace("Chasing <thing>")` just so we know for certain which ran

Reply:
very nice that is so much cleaner :-)